userspace/falco/configuration.h

```diff
@@ -161,6 +161,7 @@ class falco_configuration {
 
 	bool m_watch_config_files;
 	bool m_buffered_outputs;
+	int64_t m_falco_reload_ts;
```

Comment:
There something that feels a bit odd here. The reload timestamp is a generated value rather than a configuration per se.

I would rather put it in its own section in this file with a comment. That will make things clearer/cleaner.

Reply:
Thanks Sam, I'll take a look!

Reply:
@sgaist pushed a suggestion, let me know if you would like a different approach. Thank you!

Reply:
Looks way better ! Thanks